pkg/ottl/ottlfuncs/README.md

```diff
@@ -289,39 +289,56 @@ Unlike functions, they do not modify any input telemetry and always return a val
 
 Available Converters:
 
-- [Concat](#concat)
-- [ConvertCase](#convertcase)
-- [ExtractPatterns](#extractpatterns)
-- [FNV](#fnv)
-- [Hours](#hours)
-- [Double](#double)
-- [Duration](#duration)
-- [Int](#int)
-- [IsMap](#ismap)
-- [IsMatch](#ismatch)
-- [IsString](#isstring)
-- [Len](#len)
-- [Log](#log)
-- [Microseconds](#microseconds)
-- [Milliseconds](#milliseconds)
-- [Minutes](#minutes)
-- [Nanoseconds](#nanoseconds)
-- [Now](#now)
-- [ParseJSON](#parsejson)
-- [Seconds](#seconds)
-- [SHA1](#sha1)
-- [SHA256](#sha256)
-- [SpanID](#spanid)
-- [Split](#split)
-- [Substring](#substring)
-- [Time](#time)
-- [TraceID](#traceid)
-- [TruncateTime](#truncatetime)
-- [UnixMicro](#unixmicro)
-- [UnixMilli](#unixmilli)
-- [UnixNano](#unixnano)
-- [UnixSeconds](#unixseconds)
-- [UUID](#UUID)
+- [OTTL Functions](#ottl-functions)
```

Comment:
@etiennep for this PR please only add the new Converter name to the list.  We can have a discussion about reorganizing these docs in a separate issue.